### PR TITLE
cleanup script editor

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/script/ha-script-editor.ts [hunk 1]
```diff
@@ -178,13 +181,64 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
...
+                  slot="toolbar-icon"
+                >
+                  <mwc-button>
+                    ${this.hass.localize(
+                      "ui.panel.config.script.editor.show_trace"
+                    )}
+                  </mwc-button>
+                </a>
+              `
+            : ""}
```
Comment (piitaya) [inline src/panels/config/script/ha-script-editor.ts:241]: Should be a `mwc-list-item`

**Review by piitaya — CHANGES_REQUESTED** [review]
We can remove the footer.
Capture d’écran 2022-09-07 à 15 18 12

File: src/panels/config/script/ha-script-editor.ts (hunk 1 above)
Comment (piitaya) [reply src/panels/config/script/ha-script-editor.ts:241]: The button is included twice instead of button for desktop and list-item for mobile

Comment (bramkragten) [reply src/panels/config/script/ha-script-editor.ts:241]: Oops

File: src/panels/config/script/ha-script-editor.ts [hunk 2]
```diff
@@ -178,13 +182,64 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
...
+          <mwc-list-item
+            graphic="icon"
+            .disabled=${!this.scriptEntityId}
+            @click=${this._runScript}
+          >
+            ${this.hass.localize("ui.panel.config.script.picker.run_script")}
+            <ha-svg-icon slot="graphic" .path=${mdiPlay}></ha-svg-icon>
+          </mwc-list-item>
+
+          ${this.scriptEntityId && !this.narrow
```
Comment (piitaya) [inline src/panels/config/script/ha-script-editor.ts:228]:
```suggestion
          ${this.scriptEntityId && this.narrow
```

**Review by zsarnett — APPROVED** [review]

**Review by piitaya — APPROVED** [review]

**Pull request merged**